src/sentry/integrations/slack/tasks/find_channel_id_for_rule.py

```diff
@@ -107,7 +107,19 @@ def find_channel_id_for_rule(
 
         if rule_id:
             rule = Rule.objects.get(id=rule_id)
-            rule = Updater.run(rule=rule, pending_save=False, **kwargs)
+            rule = ProjectRuleUpdater(
+                rule=rule,
+                project=project,
+                name=kwargs.get("name"),
+                owner=kwargs.get("owner"),
+                environment=kwargs.get("environment"),
+                action_match=kwargs.get("action_match"),
+                filter_match=kwargs.get("filter_match"),
+                actions=actions,
+                conditions=kwargs.get("conditions"),
+                frequency=kwargs.get("frequency"),
+                request=kwargs.get("request"),
```

Comment:
Is there no `pending_save` parameter anymore?

Reply:
`pending_save` wasn't used anywhere in the original mediator so after converting to dataclass it was omitted here. Unless there's another scenario where that var could be passed through a mediator or something, I think it's just extraneous.